course_access_groups/models.py

```diff
@@ -4,3 +4,154 @@
 """
 
 from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
+from django.core import exceptions, validators
+from django.db import models
+from django.utils.text import slugify
+from django.utils.translation import ugettext as _
+from model_utils import models as utils_models
+from organizations.models import Organization
+from six import text_type
+
+
+def validate_domain(value):
+    """
+    Validate correct domain name.
+
+    :param value: The domain name.
+
+    This function raise ValidationError when the domain is not valid.
+    """
+    domain_regex = validators.validate_email.domain_regex
```

Comment:
why not import `validate_email` at the top of the module? This instance of `EmailValidator` will already have been loaded when the `validators` module is imported

Reply:
Good suggestion, I couldn't find a good reason why not. I refactored it.